### PR TITLE
Support cluster metadata customized tags

### PR DESCRIPTION
**Pull request by yux0**
**What changed?**
Support cluster metadata customized tags

**Why?**
Support cluster metadata customized tags

**How did you test it?**

**Potential risks**

**Is hotfix candidate?**

### CODE REVIEW

**Review by wxing1292 — APPROVED** [review]

File: service/frontend/operator_handler.go [hunk 1]
```diff
@@ -658,6 +658,7 @@ func (h *OperatorHandlerImpl) AddOrUpdateRemoteCluster(
 			InitialFailoverVersion:   resp.GetInitialFailoverVersion(),
 			IsGlobalNamespaceEnabled: resp.GetIsGlobalNamespaceEnabled(),
 			IsConnectionEnabled:      request.GetEnableRemoteClusterConnection(),
+			Tags:                     resp.GetTags(),
```
Comment (yycptt) [inline service/frontend/operator_handler.go:661]: do the same thing in admin handler?

File: proto/internal/temporal/server/api/persistence/v1/cluster_metadata.proto [hunk 2]
```diff
@@ -39,6 +39,7 @@ message ClusterMetadata {
     bool is_global_namespace_enabled = 9;
     bool is_connection_enabled = 10;
     bool use_cluster_id_membership = 11;
+    map<string,string> tags = 12;
```
Comment (yycptt) [inline proto/internal/temporal/server/api/persistence/v1/cluster_metadata.proto:42]: do we need the same thing for static config?

Comment (yux0) [reply proto/internal/temporal/server/api/persistence/v1/cluster_metadata.proto:42]: Yes. Will change it in a separate PR.

File: service/frontend/operator_handler.go (hunk 1 above)
Comment (yux0) [reply service/frontend/operator_handler.go:661]: Added

**Review by yycptt — APPROVED** [review]

**Pull request merged**